File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.py
import sys

n, m = map(int, sys.stdin.readline().split())

list1 = []
list2 = []

for i in range(n):
    a = list(map(int, sys.stdin.readline().split()))
    list1.append(a)

for i in range(n):
    a = list(map(int, sys.stdin.readline().split()))
    list2.append(a)

for i in range(n):
    x = [a + b for a, b in zip(list1[i], list2[i])]
    print(*x)

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.js
const rl = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});

let input = [];

rl.on('line', (line) => {
    input.push(line);
}).on('close', () => {
    let [n, m] = input[0].split(' ').map((a) => Number(a));
    let A = [];
    let B = [];
    let C = [];

    for (let i = 1; i <= n; i++) {
        A.push(input[i].split(' ').map((a) => Number(a)));
    }

    for (let i = n + 1; i <= 2 * n; i++) {
        B.push(input[i].split(' ').map((a) => Number(a)));
    }

    for (let i = 0; i < n; i++) {
        let tmp = [];
        for (let j = 0; j < m; j++) {
            tmp.push(A[i][j] + B[i][j]);
        }
        C.push(tmp);
    }

    C.forEach((row) => console.log(row.join(' ')));

    rl.close();
});
